common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -451,13 +485,20 @@ export class SendTransaction extends React.Component {
     }
   };
 
-  cancelTx = () => {
+  hideConfirmTx = () => {
     this.setState({ showTxConfirm: false });
   };
 
-  confirmTx = () => {
-    // TODO: Broadcast transaction
-    console.log(this.state.transaction);
+  resetState = () => {
+    this.setState({
+      to: '',
+      value: '',
+      transaction: null
+    });
+  };
```

Comment:
This name gives me the impression it resets _everything_ in state. I'd maybe call it "resetTransaction" or something.

Reply:
OK